apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -461,6 +463,7 @@ onDocumentReady(() => {
                   name="reopen_closed"
                   value="true"
                   id="reopen_closed"
+                  aria-label="reopen closed instances"
```

Comment:
Is this necessary? The `<label for="reopen_closed">` below is associated with this input and should provide the name.

Reply:
Removed in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).